README.md

```diff
@@ -356,8 +358,11 @@ The following customizations are possible:
 | deps\_dirs         | List of directories containing dependencies. It supports wildcards.                                  |
 | apps\_dirs         | List of directories containing project applications. It supports wildcards.                          |
 | include\_dirs      | List of directories provided to the compiler as include dirs. It supports wildcards.                 |
+| pa\_dirs         | List of directories containing beam files for e.g. behaviours. It supports wildcards`*`.                 |
```

Comment:
Not sure if we will go for this approach (see my comments below), but in case we do this should say 'It does not support wildcards at the moment', instead of the 'we support (not)' :)